typescript/apps/washboard-ui/src/app/components/app-lattice-client-provider.tsx

```diff
@@ -1,20 +1,38 @@
 import {
+  LatticeClientOptions,
   LatticeClient,
   LatticeClientProvider,
   NatsWsLatticeConnection,
+  LatticeConfigProvider,
 } from '@wasmcloud/lattice-client-react';
 import * as React from 'react';
 
-const client = new LatticeClient({
-  config: {
-    latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL || 'ws://localhost:4223',
-  },
-  getNewConnection: (config) =>
-    new NatsWsLatticeConnection({
-      latticeUrl: config.latticeUrl,
-    }),
-});
+type Config = Partial<LatticeClientOptions['config']>;
+const getClient = (parameters?: Config) => {
+  const config: LatticeClientOptions = {
+    config: {
+      latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL ?? 'ws://localhost:4223',
+      ...parameters,
+    },
+    getNewConnection: (c) =>
+      new NatsWsLatticeConnection({
+        latticeUrl: c.latticeUrl,
+      }),
+  };
```

Comment:
```suggestion
    getNewConnection: ({latticeUrl}) =>
      new NatsWsLatticeConnection({
        latticeUrl,
      }),
  };
```

this might be omitted here, if you want.

Reply:
Good suggestion! I will use it.

Reply:
we can go _even shorter_

```suggestion
getNewConnection: ({latticeUrl}) => new NatsWsLatticeConnection({ latticeUrl }),
```